manager/controllers/app/fybrikapplication_controller_test.go

```diff
@@ -75,10 +85,13 @@ var _ = Describe("FybrikApplication Controller", func() {
 			module := &apiv1alpha1.FybrikModule{}
 			Expect(readObjectFromFile("../../testdata/e2e/module-read.yaml", module)).ToNot(HaveOccurred())
 			moduleKey := client.ObjectKeyFromObject(module)
+      module.Namespace = controllerNamespace
 			application := &apiv1alpha1.FybrikApplication{}
+
 			Expect(readObjectFromFile("../../testdata/e2e/fybrikapplication.yaml", application)).ToNot(HaveOccurred())
 			applicationKey := client.ObjectKeyFromObject(application)
-
+			fmt.Printf("Module:  %v\n", module.Namespace)
+			fmt.Printf("Application:  %v\n", application.Namespace)
 			// Create FybrikApplication and FybrikModule
 			Expect(k8sClient.Create(context.Background(), module)).Should(Succeed())
 			Expect(k8sClient.Create(context.Background(), application)).Should(Succeed())
```

Comment:
@laurieaw63 Perhaps include a test that tries to create a fybrikapplication from a namespace that's not allowed?

Reply:
This will take some time to setup since tests needs to know when should fail or not. Would like to do this separate so can get this PR though.